scripts/pre-commit.hook

```diff
@@ -45,3 +45,11 @@ if [ "$check_enums" = "1" ]; then
     fi
     "$CHECK_EMSCRIPTEN"
 fi
+
+# Assume running from ".git/hooks" folder.
```

Comment:
A better way would be to only check modified .go files instead of running gofmt on all commits:
https://github.com/strukturag/spreed-webrtc/blob/master/src/hooks/pre-commit.hook#L6

Reply:
I decided to go with a single `gofmt.sh` file.
Using your approach would be slightly more efficient but carries additional code which will make things too complex imo.

Reply:
Yes, but running `go fmt` on all .go files for any commit is quite a lot overhead. Especially given that most changes in the repository will only affect C++ code.

Reply:
Agreed, updated again.

Reply:
Please use the whole `for file in ` block here instead of calling `gofmt.sh`.